jupyterlab_caip_optimizer/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Prefer referencing the config at the repository root.

https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/master/jupyterlab_gcpscheduler/tsconfig.json